packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -639,11 +642,47 @@ export function makeVirtualObjectManager(
   }
 
   function defineKind(tag, init, actualize, finish) {
-    return defineKindInternal(tag, init, actualize, finish, false);
+    const kindID = `${allocateExportID()}`;
+    return defineKindInternal(kindID, tag, init, actualize, finish, false);
+  }
+
+  let kindIDID;
+  const kindDescriptors = new WeakMap();
+
+  function reanimateDurableKindID(vobjID, _proforma) {
+    const { subid: kindID } = parseVatSlot(vobjID);
+    const raw = syscall.vatstoreGet(`vom.kind.${kindID}`);
+    assert(raw, X`unknown kind ID ${kindID}`);
+    const durableKindDescriptor = JSON.parse(raw);
+    const kindToken = Far('kind', {});
+    kindDescriptors.set(kindToken, durableKindDescriptor);
+    return kindToken;
+  }
+
+  function makeKindHandle(tag) {
+    if (!kindIDID) {
+      kindIDID = `${allocateExportID()}`;
+      syscall.vatstoreSet('kindIDID', kindIDID);
+      vrm.registerKind(kindIDID, reanimateDurableKindID, () => null, true);
+    }
+    const kindID = `${allocateExportID()}`;
+    const kindIDvref = `o+${kindIDID}/${kindID}`;
+    const durableKindDescriptor = { kindID, tag };
+    const kindHandle = Far('kind', {});
```

Comment:
```suggestion
    const kindHandle = makeHandle('Kind');
```
though feel free to ignore this until we put `makeHandle` in a more reusable place.

Reply:
or, should the handle's farName include the tag?

```suggestion
    const kindHandle = Far(`${tag} kind`, {});
```
or
```suggestion
    const kindHandle = makeHandle(`${tag} kind`);
```

Reply:
I'm going to ignore all this `makeHandle` stuff until `makeHandle` is a thing.

Reply:
> or, should the handle's farName include the tag?

I don't think so.  That would be conflating value and type in a way I find confusing.